packages/server/.gitignore

```diff
@@ -21,3 +24,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+config.yaml
```

Comment:
@thesocialdev let's start adding newline at EOF on the next PRs

Reply:
Done.